### DISCUSSION TITLE
sd3 dreambooth VRAM

### DISCUSSION THREAD

**Discussion by carolineys** [huggingface/diffusers#8848] [Q&A]
Anyone know what is the min vram required to do dreambooth or dreambooth with lora for sd3? I tried to run the script on T4, but it says "allocate memory failed".

**Comment by asomoza** [comment]
@sayakpaul did [this](https://github.com/huggingface/diffusers/blob/main/examples/research_projects/sd3_lora_colab/sd3_dreambooth_lora_16gb.ipynb) awesome jupyter notebook that you can run on google collab to train SD3 (dreambooth lora) under 16GB of VRAM. It all depends on how you do it, if you compute the embeddings before the training you can save a lot of VRAM.

You can read more about it [here](https://github.com/huggingface/diffusers/tree/main/examples/research_projects/sd3_lora_colab)

**Comment by Darth-Zzz** [comment]
Same question here.
